test/agentchat/test_agent_file_logging.py

```diff
@@ -0,0 +1,111 @@
+import json
+import os
+import sys
+import uuid
+
+import pytest
+
+sys.path.append(os.path.join(os.path.dirname(__file__), ".."))
+sys.path.append(os.path.join(os.path.dirname(__file__), "../.."))
+
+from conftest import skip_openai  # noqa: E402
+from test_assistant_agent import KEY_LOC, OAI_CONFIG_LIST  # noqa: E402
+
+import autogen
+import autogen.runtime_logging
+from autogen.logger.file_logger import FileLogger
+
+
+@pytest.fixture
+def logger() -> FileLogger:
```

Comment:
Using default for tests can cause overwriting non-test logs when running tests from the same environment. 

Please use temporary file `tempfile` for testing (and remember to clean up).

Reply:
yeah working on it.

Reply:
having used the `tempfile`, facing a concurrency issue. looks like issue with threading in python. @ekzhu